coffeecard/CoffeeCard.Library/Services/v2/TicketService.cs

```diff
@@ -46,14 +45,18 @@ public async Task IssueTickets(Purchase purchase)
 
         public Task<List<TicketResponse>> GetTickets(User user, bool includeUsed)
         {
-            return _context.Tickets.Where(t => t.Owner.Equals(user) && t.IsUsed == includeUsed).Include(t => t.Purchase)
+            return _context.Tickets
+                .Where(t => t.Owner.Equals(user) && t.IsUsed == includeUsed)
+                .Include(t => t.Purchase)
+                .Include(t => t.UsedOnMenuItem)
                 .Select(t => new TicketResponse
                 {
                     Id = t.Id,
                     DateCreated = t.DateCreated,
                     DateUsed = t.DateUsed,
                     ProductId = t.ProductId,
-                    ProductName = t.Purchase.ProductName
+                    ProductName = t.Purchase.ProductName,
+                    UsedOnMenuItemName = t.UsedOnMenuItem != null ? t.UsedOnMenuItem.Name : null
```

Comment:
Suggest to use the [Null Conditional Operator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-)

```suggestion
UsedOnMenuItemName = t.UsedOnMenuItem?.Name
```

Reply:
I tried that, but it doesn't compile:

> An expression tree lambda may not contain a null propagating operator. [(CS8072)](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/expression-tree-restrictions?f1url=?appId=roslyn&k=k(CS8072))

Oddly enough dropping the question mark _does_ compile, however I would not do this.

Reply:
Alright, let's close this point then :)